### PR TITLE
OCP4: Update k8s action to build image on new PR

### PR DESCRIPTION
**Pull request by Vincent056**
This triggers the K8S image action to run on every new PR created

### CODE REVIEW

**Comment by Vincent056** [comment]
I think this worked: ghcr.io/complianceascode/k8scontent:11384

**Review by yuumasato — APPROVED** [review]
LGTM.

I was able to deploy the content by doing: `CONTENT_IMAGE=ghcr.io/complianceascode/k8scontent:11384 make deploy-local`

And how about, for visibility purposes, posting a comment to the PR?
For example, the final tasks of this workflow post a comment to the PR:
https://github.com/ComplianceAsCode/content/blob/41ad1904aa2cdb1b3359f13563bdf5e9b3cb33db/.github/workflows/compare-ds.yaml#L90

File: .github/workflows/k8s-content-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    name: Comment on the PR
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            :robot: The image for this PR is available at ghcr.io/complianceascode/k8scontent:${{ needs.get-pr-number.outputs.pr-number }}
```
Comment (yuumasato) [inline .github/workflows/k8s-content-pr.yaml:52]: @Vincent056 The test for  EOF failed. Please ensure  there is newline at the end of the file.

Comment (yuumasato) [inline .github/workflows/k8s-content-pr.yaml:52]: I suggest the comment is updated or inserted, and a formatting that makes it easy to copy the URL.
```suggestion
          message: |
            :robot: The image for this PR is available at:
            `ghcr.io/complianceascode/k8scontent:${{ needs.get-pr-number.outputs.pr-number }}`
          comment_tag: kubernetes_content_image
```

**Review by yuumasato — CHANGES_REQUESTED** [review]
@Vincent056 One more  round and should be excelent

File: .github/workflows/k8s-content-pr.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            :robot: The image for this PR is available at:
+            `ghcr.io/complianceascode/k8scontent:${{ needs.get-pr-number.outputs.pr-number }}`
+          comment_tag: kubernetes_content_image
```
Comment (Vincent056) [inline .github/workflows/k8s-content-pr.yaml:54]:
```suggestion
          comment_tag: kubernetes_content_image

```
Adding a new line to EOF

**Review by yuumasato — APPROVED** [review]
Looking great!

**Comment by xiaojiey** [comment]
/lgtm
tried before commands to deploy pb with the custom content image
"export CONTENT_IMAGE=ghcr.io/complianceascode/k8scontent:11384
# make images && make push && make catalog && make catalog-deploy
make images && make push && make deploy"
$ oc get pb
NAME     CONTENTIMAGE                                CONTENTFILE         STATUS
ocp4     ghcr.io/complianceascode/k8scontent:11384   ssg-ocp4-ds.xml     VALID
rhcos4   ghcr.io/complianceascode/k8scontent:11384   ssg-rhcos4-ds.xml   VALID

**Comment by yuumasato** [comment]
Test `Gate / Build, Test on Debian 10 (Container) (push) ` has been failing consistently, although I see it passing in other PRs.
This patch doesn't affect Debian builds in any way, so I'm merging.
Thank you @Vincent056

**Pull request merged**